app/js/showElement.js

```diff
@@ -0,0 +1,91 @@
+/* eslint "no-unused-expressions":  "off" */
+const fs = parent.require('fs');
```

Comment:
parent is not defined. You probably want `const parent = window.parent` before these lines